index.html

```diff
@@ -0,0 +1,250 @@
+            <p class="description lato-font text-white ms-3">A glance of the different cultures of Australia, Melanesia, Polynesia & Micronesia.</p>
+          </div>
+
+          <!--Oceania | South Pacific-->
+          <div class="section-2-1-4-desktop mb-3 me-3 px-3 py-4 d-md-flex flex-md-column align-items-md-center">
+            <img src="images/2078330611551682368.svg" alt="A tropical island." class="section-2-1-icon44 mb-5 img-fluid" id="section-2-1-icon44"/>
+            <p class="title lato-font font-orange fw-bold mt-4 me-2">Oceania</p>
+            <p class="description lato-font text-white ms-3">Discover the beauty of the tropical islands of Oceania.</p>
+          </div>
+
+          <!--Abstract-->
+          <div class="section-2-1-5-desktop mb-3 me-3 px-3 py-4 d-md-flex flex-md-column align-items-md-center">
+            <img src="images/2962405911631503234.svg" alt="An abstract design." class="section-2-1-icon55 mb-1 img-fluid" id="section-2-1-icon55"/>
+            <p class="title lato-font font-orange fw-bold mt-5 pt-5 ms-1">Abstract</p>
+            <p class="description lato-font text-white ms-3">Find some abstract pieces here.</p>
+          </div>
+        </div>
+
+        <div class="d-md-flex flex-md-column align-items-md-center mt-3">
+          <a href="#" class="contact-us-btn lato-font bg-orange fw-bold text-center text-decoration-none border-orange col-md-10 px-1 py-4">Contact me</a>
+        </div>
+      </section>
+
+      <!--Section 4 | Partners for mobile screens-->
+      <section class="section-4-mobile row d-flex d-sm-flex d-md-none px-2 py-5 bg-darkgrey">
+        <h2 class="heading lato-font font-darkgrey text-white text-center fw-bold col-12 col-sm-12">Partners</h2>
+
+        <div class="section-4-1-mobile col-12 col-sm container-fluid mt-5 ps-5">
+          <div class="row">
+            <img src="images/National_Geographic-Logo.wine.svg" alt="National Geographic logo" class="national-geographic-logo col-6 col-sm-6"/>
+            <img src="images/Google-Logo.wine.svg" alt="Google logo" class="google-logo col-6 col-sm-6"/>
+          </div>
+
+          <div class="row mt-4">
+            <img src="images/tourism-fiji-seeklogo.com.svg" alt="Tourism Fiji logo" class="tourism-fiji-logo col-4 col-sm-4"/>
+            <img src="images/westin-1.svg" alt="The Westin logo" class="the-westin-logo col-4 col-sm-4"/>
+            <img src="images/243346.svg" alt="Fiji Water logo" class="fiji-water-logo col-4 col-sm-4"/>
+          </div>
+        </div>
+      </section>
+
+      <!--Section 4 | Partners for tablet & desktop screens-->
+      <section class="section-4-desktop row d-none d-sm-none d-md-flex px-2 py-5 bg-darkgrey">
+        <h2 class="heading lato-font font-darkgrey text-white text-center fw-bold col-md-12">Partners</h2>
+
+        <div class="section-4-1-desktop col-md-12 d-md-flex flex-md-row justify-content-ms-around mt-5 ps-5">

```

Comment:
- [ ] Kindly fix the image.
![Capturecf](https://user-images.githubusercontent.com/81444338/175099978-75274143-4aba-4126-a3b1-9ac9e26b8432.PNG)